genesis/types.go

```diff
@@ -65,18 +65,24 @@ func (mm *MultisigMeta) ActorMeta() json.RawMessage {
 	return out
 }
 
-type Actor struct {
-	Type    ActorType
-	Balance abi.TokenAmount
+type BindMiner struct {
+	SealProof abi.RegisteredSealProof
+	Address   address.Address
+}
 
-	Meta json.RawMessage
+type Actor struct {
+	Type       ActorType
+	Balance    abi.TokenAmount
+	BindMiners []BindMiner
```

Comment:
A bit odd to put it here, I'd probably just add the Address field to the Miner struct above. Or just change the template to
```go
type Template struct {
	Accounts map[string]Actor // key has to be string because json.. should be set as address string (e.g `t0123`)
	Miners   map[string]Miner

...
}
```

(though that will require a bit more significant refactoring)